packages/mobile/src/navigator/types.tsx

```diff
@@ -141,6 +141,7 @@ export type StackParamList = {
   [Screens.PincodeEnter]: {
     withVerification?: boolean
     onSuccess: (pin: string) => void
+    onCancel?: () => void
```

Comment:
Why not making this param mandatory? 
This would enforce that all places using `PincodeEnter` handle the cancel case,